packages/core/src/useCheckbox/useCheckbox.ts

```diff
@@ -307,6 +311,25 @@ export function useCheckbox<TValue = string>(
     },
     field as FormField<TValue>,
   );
+
+  if (__DEV__) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    registerCheckboxWithDevtools({ ...exposedField, ...field } as any, form?.id);
+
+    watch(
+      () => ({
+        errors: errorMessage.value,
+        isValid: !errorMessage.value,
+        value: fieldValue.value,
+      }),
+      refreshInspector,
+      {
+        deep: true,
+      },
+    );
```

Comment:
Since you are passing the entire thing to `registerCheckboxWithDevtools` do we need to handle the refresh here? Maybe `registerCheckboxWithDevtools` should handle it.